pkg/data/components/DsymSymbolicationProcessor.yaml

```diff
@@ -0,0 +1,100 @@
+kind: DsymSymbolicationProcessor
+name: Symbolicate Swift/iOS Errors
+style: processor
+type: base
+status: development
+version: v0.1.0
+summary: Symbolicate stack traces from iOS/Swift applications using dSYM symbol files
+description: |
+  Symbolicate iOS/Swift stack traces using dSYM symbol files.
+  Supports dSYM files stored in AWS S3 or Google Cloud Storage.
+tags:
+  - category:processor
+  - service:collector
+  - signal:OTelTraces
+  - input:Traces
+  - output:Traces
+ports:
+  # inputs
+  - name: Traces
+    direction: input
+    type: OTelTraces
+  # outputs
+  - name: Traces
+    direction: output
+    type: OTelTraces
+properties:
+  - name: SymbolStore
+    display: Symbol Store
+    summary: Select the service where symbol files are stored
+    description: |
+      Select the service where symbol files are stored.
+      Options are `AWSS3` and `GoogleCloudStorage`.
+      The default is `AWSS3`.
+    type: string
+    subtype: oneof(AWSS3, GoogleCloudStorage)
```

Comment:
Does HPSF support these options having spaces? The card calls for `AWS S3` and `Google Cloud Store`.